### PR TITLE
unofficial-fhict-document-template:0.10.0

### PR DESCRIPTION
**Pull request by TomVer99**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This package is a document template for FHICT. It makes sure the user keeps to a style and does some chores for the user.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

*note:* The original GIT repo's version will be synced with this (0.10.0).
*note:* The use of the Fontys logo is within specified guidelines by Fontys themselves.

### CODE REVIEW

**Comment by TomVer99** [comment]
Oops i forgot something in the README...

**Comment by TomVer99** [comment]
Fixed

File: packages/preview/unofficial-fhict-document-template/0.10.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#import "@preview/unofficial-fhict-document-template.typ:0.10.0": *
```
Comment (elegaanz) [inline packages/preview/unofficial-fhict-document-template/0.10.0/template/main.typ:1]: The .typ extension shouldn't be there:

```suggestion
#import "@preview/unofficial-fhict-document-template:0.10.0": *
```

**Comment by elegaanz** [comment]
Thank you for the fix :)

**Comment by TomVer99** [comment]
> Thank you for the fix :)

Thank you for the review, its much appreciated 😄

**Pull request merged**